### ISSUE TITLE
Steam client resets scroll position in the Library list on various occasions

### ISSUE THREAD

**Issue by Lastique** [ValveSoftware/steam-for-linux#9604]
#### Your system information

* Steam client version (build number or date): 1686779606
* Distribution (e.g. Ubuntu): Kubuntu 22.04
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:

In the Steam Library, scroll position is often lost in the list of games on the left. Meaning that regardless of the currently selected item, the list often resets back to the default scroll position, showing the first items.

#### Steps for reproducing this issue:

Scenario 1:

1. Start steam client. In my case, Steam Library is the starting page.
2. Scroll down the list of games and click on one. The list resets back to the initial scroll position.

Scenario 2:

1. In Steam library, scroll down the list of games and click on one. Make sure the list stays at the scroll position that shows the selected game (i.e. if the list reset itself as in Scenario 1 above, scroll back down).
2. Go to Downloads by clicking Manage Downloads below.
3. On the Downloads screen, click on Library at the top (in the Store Library Community... list). The list scroll position resets.

**Comment by Lastique** [comment]
> One workaround is to press one of the up/down arrow keys. That will force the library listing to re-center itself on the next/previous item.

This works... weirdly. If I press down, the scroll position is set so that the item that is next after the selected one is at the top of the list, and the selected item is nowhere to be seen. Similarly, if I press up, the item prior to the selected one is now at the bottom - again, making the selected item not visible. Yes, this kind of helps to get closer to the selected item, but I hardly find this a good UX.

**Comment by eukara** [comment]
Agreed. I have a massive game library and so that janky workaround is better than nothing at all.

**Comment by tastypurgen** [comment]
Can't believe this issue still exists. It's really annoying on steam deck
